src/core/helpers.ts

```diff
@@ -41,7 +41,7 @@ export function parseNodeAliases(input: string): NodeAliases {
 
 export function splitFlagInput(input: string, separator = ',') {
   if (typeof input !== 'string') {
-    throw new SoloError(`input [input='${input}'] is not a comma separated string`);
+    return [];
```

Comment:
did you check to see the impact of other places where this was used?  it might have been a prompt to let the user know they formatted it wrong.

if this doesn't impact anything else it is fine, else you might need to change the other locations to throw this error if empty array, or change your new reference to catch the error and set it to an empty array.

Reply:
All other uses have a check for an empty input before calling `splitFlagInput`. I modified it again and now the other uses shouldn't be affected in any way.